undercloud.yml

```diff
@@ -50,6 +50,7 @@
           --version {{ osp_release }} \
           --build {{ osp_puddle }} \
           --images-task rpm \
+          --hieradata-config "ironic::conductor::default_boot_mode=bios,ironic::drivers::ilo::default_boot_mode=bios" \
```

Comment:
`ilo` driver is specific to HPE based server. Is there any reason we don't use `ipmi`?

Reply:
It refers to the section in ironic.conf, default_boot_mode is mentioned at 2 places in the file.